src/migrations/20230927071830-reset-pnps-feedback.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+exports.up = function (db, callback) {
+    db.runSql(
+        `
+       DELETE FROM user_feedback WHERE feedback_id = 'pnps';
```

Comment:
Maybe only when it is older than 3 months?

Reply:
Unfortunately, this table does not have any timestamp columns.

Reply:
There is column `given`, with timestamp.

Reply:
> There is column `given`, with timestamp.

Yes, but that is only populated if you gave feedback. If you hide the feedback form it won't be populated.

Reply:
So two options: 
* We clear all entries
* We clear only the entries that have previously given feedback

What do you prefer?